docs/server/upgrading.rst

```diff
@@ -3,170 +3,222 @@
+As we haven't started the RQ worker yet we will run the commands using the
+``--no-rq`` option:
 
-* We highly recommended that you use a virtual environment. If your install
-  currently doesn't use one then please :ref:`set up a virtualenv
-  <installation#setup_environment>`.
+.. code-block:: bash
 
-* If you are upgrading from a version older than Pootle 2.7.0, then you must
-  first upgrade to Pootle 2.6.0.
+   (env) $ pootle calculate_checks --no-rq
+   (env) $ pootle refresh_stats --no-rq
 
-  .. code-block:: bash
+These commands may take some time to complete.
 
-     (env) $ pip install --upgrade "Pootle>=2.6,<2.7"
-     (env) $ pootle setup
 
-  Then continue with the upgrade process.
+.. _upgrading#setup-users:
 
-* Upgrade the Pootle package:
+Set up users
+------------
 
-  .. code-block:: bash
+Any accounts that do not have an email address registered will not be able to
+log in. You can set the email for a user using the :djadmin:`update_user_email`
+command.
 
-     (env) $ pip install --upgrade Pootle
+For example to set the email for user "admin" to "admin@example.com": 
 
-* Update your custom Pootle settings to adjust for any changes and to include
-  any new settings. Delete any obsolete settings. Check the :ref:`available
-  settings <settings#available>` as needed.
+.. code-block:: bash
 
-  .. note:: Running :djadmin:`pootle check` will highlight settings that have
-     been made obsolete or renamed.
+   (env) $ pootle update_user_email admin admin@example.com
 
-  .. note:: If you are upgrading from a version of Pootle that uses
-     :file:`localsettings.py` then you must :ref:`move your custom settings
-     <settings#customizing>` to a new location in order to ensure that Pootle
-     uses them.
 
-* Perform the database schema and data upgrade by running:
+As of Pootle 2.7 users must now verify their email in order to log in.
 
-  .. code-block:: bash
+You can automatically verify a user bypassing email verification using the
+:djadmin:`verify_user` command.
 
-     (env) $ pootle migrate
+For example to automatically verify the admin user:
 
+.. code-block:: bash
 
-* Reapply your custom changes to Pootle code, templates or styling. Read about
-  :doc:`customization of style sheets and templates
-  </developers/customization>` to adjust your customizations to the correct
-  locations and approach in the new release.
+   (env) $ pootle verify_user admin
 
-  .. note:: If you have customized the CSS styling or the JavaScript code you
-     will have to run the following commands to update the static assets:
+If you wish to automatically verify all of your existing users:
 
-     .. code-block:: bash
+.. code-block:: bash
 
-       (env) $ cd $pootle_dir/pootle/static/js/
-       (env) $ npm install
-       (env) $ npm update
-       (env) $ pootle webpack
-       (env) $ pootle collectstatic --noinput --clear -i node_modules -i *.jsx
-       (env) $ pootle assets build
+   (env) $ pootle verify_user --all
```

Comment:
I'd point them to the ``--all`` option in the command rather then hinting towards doing this here.  Its seriously risky on a large install and undoes any of the good verification has.